addons/website_forum/data/forum_default_faq.html

```diff
@@ -1,63 +1,84 @@
-<div id="accordion">
-    <div class="card mb-2 bg-100">
-        <a href="#collapseOne" class="card-header" id="headingOne" data-toggle="collapse" aria-expanded="false" aria-controls="collapseOne">
-            <b>What kinds of questions can I ask here?</b> <i class="fa fa-caret-down float-right"/>
-        </a>
-        <div id="collapseOne" class="collapse" aria-labelledby="headingOne" data-parent="#accordion">
-            <div class="card-body">
-                <p>This community is for professional and enthusiast users, partners and programmers. You can ask questions about:</p>
-                <ul>
-                    <li>how to install Odoo on a specific infrastructure,</li>
-                    <li>how to configure or customize Odoo to specific business needs,</li>
-                    <li>what's the best way to use Odoo for a specific business need,</li>
-                    <li>how to develop modules for your own need,</li>
-                    <li>specific questions about Odoo service offers, etc.</li>
-                </ul>
-                <p><b>Before you ask - please make sure to search for a similar question.</b> You can search questions by their title or tags. It’s also OK to answer your own question.</p>
-                <p><b>Please avoid asking questions that are too subjective and argumentative</b> or not relevant to this community.</p>
+<section class="s_faq_collapse pt32 pb32">
```

Comment:
as discuss, should be a render with karma link in 4th accordon

Reply:
I though I would keep the html file. I didn't understand it that way, so you want me to have a template and to render it into html ? It's that right ?

Reply:
:+1:

Reply:
Like I already told you to try and you told me you tried and that it did not work... don't you remember?

Reply:
Yes, but I didn't remember why, so I tried it again (it was fast to do it), and so, it feels like I enter into the compute method before knowing that the template exists (During the -i).
If I put a debug line in the compute and try to find any template in website_forum, I won't find them.
So, I guess I keep the html file ?

Reply:
You still need to have a specific link by forum, so it cannot be just HTML...

@JKE-be should we ask RCO if it is possible somehow to call render_template in a default method (maybe there is a way to mark a template to be created first?). Or we could still post-process the HTML and replace the "href="<default-link-to-faq>" we'd put in HTML by the right URL with a search/replace? If you find that acceptable. Still would prefer that to create a generic link IMHO, what do you think?

Reply:
If I put a debug line in the compute and try to find any template in website_forum, I won't find them.

How do you want to pass in your debugger while you don't create a record...
you need to have the template created before the call to 
/addons/website_forum/data/forum_data.xml#L4

and it should work imo

Reply:
Should I create a template or a record ? I tried some things yesterday but didn't get any result :/

Reply:
@Esperax A template is a record...

You need the view created before the forum creation. So:
- The file containing the view creation should be loaded first
- The file containing the forum creation should be loaded last
- Or maybe create them both in the same file? To check with the current structure of website_forum files